_unittest/test_35_MaxwellCircuit.py

```diff
@@ -114,3 +114,11 @@ def test_08_import_netlist(self):
         self.aedtapp.insert_design("SchematicImport")
         self.aedtapp.modeler.schematic.limits_mils = 5000
         assert self.aedtapp.create_schematic_from_netlist(self.netlist_file1)
+
+    def test_09_create_voltage_source(self):
+        id = self.aedtapp.modeler.schematic.create_vsin(name="voltage_source", value=3.14, angle=90)
+        assert id.parameters["Va"] == "3.14"
+
+    def test_10_create_current_source(self):
+        id = self.aedtapp.modeler.schematic.create_isin(name="current_source", value=2.72, angle=90)
```

Comment:
@DaveTwyman same here.

Reply:
@gmalinve I'm sure all usages of id have already been replaced with component. Is this an older branch you've reviewed perhaps?

Reply:
@DaveTwyman what do you mean an older branch? this is your branch.

Reply:
@gmalinve , I should have wrote 'older commit' not 'older branch', I checked and id is replaced with component everywhere.

Reply:
@DaveTwyman please look up, fix the tests where there is id and not component. You change from id to component only in the code, not in the tests.